src/Goerli-DssSpell.t.sol

```diff
@@ -458,4 +533,537 @@ contract DssSpellTest is GoerliDssSpellTestBase {
 
         assertEq(vest.rxd(1), WAD);
     }
+
+    function testSpellIsCast_RWA009_INTEGRATION_BUMP() public {
```

Comment:
Any reason why not keeping the same 8 then 9 order?

Reply:
I wouldn't say for this spell but from the next one I think we should have a specific RWA test file.

Reply:
No particular reason, but I'll reorder them.

Reply:
resolving...